lib/manager/bundler/extract.ts

```diff
@@ -189,6 +189,7 @@ export async function extractPackageFile(
       for (const dep of res.deps) {
         const lockedDepValue = lockedEntries.get(dep.depName);
         if (lockedDepValue) {
+          dep.depType = 'dependencies';
```

Comment:
I think this should outside the condition.

Reply:
I'm not sure where exactly it should be, it could be in a different area even, maybe here:
    if (gemMatch) {
      const dep: PackageDependency = {
        depName: gemMatch.groups.depName,
        managerData: { lineNumber },
      };
      if (gemMatch.groups.currentValue) {
        const currentValue = gemMatch.groups.currentValue;
        dep.currentValue = regEx(/\s*,\s*/).test(currentValue)
          ? currentValue
          : currentValue.slice(1, -1);
      }
      dep.datasource = RubyGemsDatasource.id;
      {here?}
      res.deps.push(dep);
    }